curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/concatenating-strings-with-the-plus-equals-operator.md

```diff
@@ -24,7 +24,7 @@ ourStr += "I come second.";
 
 # --instructions--
 
-Build `myStr` over several lines by concatenating these two strings: `This is the first sentence. ` and `This is the second sentence.` using the `+=` operator. Use the `+=` operator similar to how it is shown in the example. Start by assigning the first string to `myStr`, then add on the second string.
+Build `myStr` over several lines by concatenating these two strings: <pre>`This is the first sentence. `</pre> and `This is the second sentence.` using the `+=` operator. Use the `+=` operator similar to how it is shown in the example. Start by assigning the first string to `myStr`, then add on the second string.
```

Comment:
@nhcarrigan would you say something similar here too? adding the "Be sure to add a space between the two strings" hre too?

Reply:
Yep.